cypress/integration/networking.spec.ts

```diff
@@ -124,12 +124,6 @@ context("p2p networking", () => {
           commands
             .pickWithContent(["followed-peers", "peer-abbey"], shortenedPeerId)
             .should("exist");
-          commands
-            .pickWithContent(
-              ["followed-peers", "peer-abbey", "follow-toggle"],
-              "Following"
-            )
-            .should("exist");
```

Comment:
We can ignore the content but we should still check the toggle.
```ts
commands.pick("followed-peers", "peer-abbey", "follow-toggle").should("exist")
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1928/commits/6e8a1b09c376d1fb368842365054e938de5aefe7